src/label/Label.ts

```diff
@@ -1,85 +1,61 @@
 import { WidgetBase } from '@dojo/widget-core/WidgetBase';
-import { DNode, SupportedClassName } from '@dojo/widget-core/interfaces';
+import { DNode } from '@dojo/widget-core/interfaces';
 import { ThemedMixin, ThemedProperties, theme } from '@dojo/widget-core/mixins/Themed';
 import { v } from '@dojo/widget-core/d';
-import { assign } from '@dojo/core/lang';
 import * as css from './styles/label.m.css';
 import * as baseCss from '../common/styles/base.m.css';
 
-/**
- * Label settings for form label text content, position (before or after), and visibility
- */
-export interface LabelOptions {
-	content: string;
-	before?: boolean;
-	hidden?: boolean;
-}
-
-/**
- * Default settings for form labels
- */
-const labelDefaults = {
-	content: '',
-	before: true,
-	hidden: false
-};
-
 /**
  * @type LabelProperties
  *
  * Properties that can be set on a Label component
  *
  * @property forId     ID to explicitly associate the label with an input element
- * @property label      Label settings for form label text, position, and visibility
+ * @property disabled
```

Comment:
Were description for these properties intentionally omitted? It's probably fine if so, just making sure.

Reply:
they seemed kinda obvious to me. Can add them in if we feel it's important